.github/workflows/test.yml

```diff
@@ -0,0 +1,36 @@
+
+name: build
+
+on: push
+
+jobs:
+
+  test:
+
+    name: Test on Ruby ${{ matrix.ruby }}
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        ruby:
+          - 2.1
+          - 2.2
+          - 2.3
+          - 2.4
```

Comment:
Is now a good time to remove support for EOL rubies?

Reply:
(I've been doing the same for `Rack::ECG`)

Reply:
This library isn't using any new features introduced in later Rubies. Until we need to do so, I don't have a strong incentive to remove support.